Newsletters: add 219 (2022-09-28)

harding opened the pull request:
- [x] Lede, releases/RCs, topic links @harding
- [x] Stack Exchange section @bitschmidty

Xekyo commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -67,7 +67,30 @@ answers posted since our last update.*
+  Murch notes that the gap in the timeline could be explained by the genesis
+  block having a higher difficulty target than required, Satoshi setting the
+  block's timestamp in the past, or the [original Bitcoin software waiting][github
+  jrubin annotated] for a peer before continuing to mine.
```
```suggestion
  jrubin annotated] for a peer before starting to mine.
```

Xekyo reviewed (commented):
Just did a quick pass

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+    channel, including reducing it to zero.  Although this can be
+    dangerous---the lower the reserve, the less penalty there is for
+    cheating---it can be useful for certain situations.  Most notably,
+    setting the reserve can allow the remote peer to spend all of their
```
(check me on this!)
```suggestion
    setting the reserve to zero can allow the remote peer to spend all of their
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+    Developer ZmnSCPxj [described][zmnscpxj ratecards] a simple way to
+    use ratecards, "you can model a rate card as four separate channels
+    between the same two nodes, with different costs each.  If the path
+    at the lowest cost fails, you just try at another route that may
```
```suggestion
    at the lowest cost fails, you just try another route that may
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+
+    The proposal also allows for negative fees.  For example, a channel
+    could subsidize payments forwarded through it when it had more than
+    75% outbound capacity by adding 1 satoshi to every 1,000 satoshi in
```
```suggestion
    75% outbound capacity by adding 1 satoshi to every 1,000 satoshis in
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+    frequently used for receiving payments.
+
+    Neigut notes that some nodes are already adjusting their fees based
+    on channel capacity, and that fee ratecards provides a more
```
```suggestion
    on channel capacity, and that fee ratecards provide a more
```
Or maybe "and that fee ratecards feature provides a more ..."

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+  Anthony Towns [posted][towns bi] to the Bitcoin-Dev mailing list a
+  description of a fork of Bitcoin Core he's working on that will only
+  operate on the default [signet][topic signet] and which will enforce
+  rules for soft fork proposals with high quality specification and
```
```suggestion
  rules for soft fork proposals with high-quality specifications and
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+
+- [Is it possible to fingerprint HD wallets?]({{bse}}115311)
+  Pieter Wuille points out that while identifying UTXOs created using an [HD
+  wallet][topic bip32] is not possible, other onchain data can be used to
```
```suggestion
  wallet][topic bip32] is not possible, other on-chain data can be used to
```

harding replied:
We previously settled on "onchain", see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+- [Is it possible to fingerprint HD wallets?]({{bse}}115311)
+  Pieter Wuille points out that while identifying UTXOs created using an [HD
+  wallet][topic bip32] is not possible, other onchain data can be used to
+  fingerprint wallet software including type of inputs used, types of outputs
```
```suggestion
  fingerprint wallet software including types of inputs used, types of outputs
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+  created, order of inputs and outputs in the transaction, [coin
+  selection][topic coin selection] algorithm, and use of [timelocks][topic timelocks].
+
+- [Why is there a 5 day gap between the genesis block and block 1?]({{bse}}115344)
```
```suggestion
- [Why is there a 5-day gap between the genesis block and block 1?]({{bse}}115344)
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+  GUI, defaulting to RBF in RPCs, and using [`mempoolfullrbf`][news208
+  mempoolfullrbf] to allow replacements without signaling.
+
+- [Why would I need to ban peer nodes in Bitcoin network?]({{bse}}115183)
```
```suggestion
- [Why would I need to ban peer nodes in the Bitcoin network?]({{bse}}115183)
```

LarryRuane commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+  In contrast to [discouraging a peer][bitcoin 23.x banman], user RedGrittyBrick
+  explains that a node operator could choose to manually ban a peer using the
+  [`setban`][setban rpc] RPC if the peer is misbehaving, a suspected malicious or surveillance
+  node, or part of cloud provider's network, among other reasons.
```
```suggestion
  node, or part of a cloud provider's network, among other reasons.
```

LarryRuane reviewed (commented):
LGTM, suggesting a few minor grammar tweaks.

bitschmidty commented on _posts/en/newsletters/2022-09-28-newsletter.md:
```diff
@@ -0,0 +1,150 @@
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is it possible to fingerprint HD wallets?]({{bse}}115311)
+  Pieter Wuille points out that while identifying UTXOs created using an [HD
+  wallet][topic bip32] is not possible, other onchain data can be used to
+  fingerprint wallet software including type of inputs used, types of outputs
+  created, order of inputs and outputs in the transaction, [coin
+  selection][topic coin selection] algorithm, and use of [timelocks][topic timelocks].
```
Reworded the question, but took some liberties

glozow reviewed (commented):
LGTM, awesome newsletter this week

renepickhardt commented:
Not time sensitive, but today I received a decent answer to a two year old fairly upvoted and until today unanswered question on stack exchange https://bitcoin.stackexchange.com/a/115385/81940 thought I notify you about it.

harding commented:
Made all suggested edits (thanks for reviews @LarryRuane @Xekyo @glozow ), added releases/RCs, added topic links, and reviewed @bitschmidty's section (thanks!).  Thanks everyone!

(Also upvoted the answer to @renepickhardt's SE question, since it was good, so hopefully it ranks high enough for next month's SE section.)

bitschmidty commented:
Squashed and merged, thank you @harding @Xekyo @LarryRuane @glozow for contributing and reviewing this week!

bitschmidty commented:
@renepickhardt thanks for the suggestion for the stack exchange, will consider it for next month!
